### ISSUE TITLE
[Accessibility] Typing does not work for Chat variable (#) in Copilot Chat

### ISSUE THREAD

**Issue by jooyoungseo** [microsoft/vscode#245125]
Type: <b>Bug</b>

In Copilot Chat, participant `@` and command `/` accept user input in their suggestion popup such that users can keep typing command name to quickly search.

However, Chat variable `#` doesn't allow users to type in its popup which means users need to manually ArrowDown/ArrowUp to find their desired variable.

VS Code version: Code - Insiders 1.99.0-insider (99c9c6c8eb0aef3fce659b0fac1ff3130c4e34a4, 2025-03-28T21:28:51.696Z)
OS version: Windows_NT x64 10.0.26100
Modes:

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|13th Gen Intel(R) Core(TM) i7-13650HX (20 x 2803)|
|GPU Status|2d_canvas: enabled<br>canvas_oop_rasterization: enabled_on<br>direct_rendering_display_compositor: disabled_off_ok<br>gpu_compositing: enabled<br>multiple_raster_threads: enabled_on<br>opengl: enabled_on<br>rasterization: enabled<br>raw_draw: disabled_off_ok<br>skia_graphite: disabled_off<br>video_decode: enabled<br>video_encode: enabled<br>vulkan: disabled_off<br>webgl: enabled<br>webgl2: enabled<br>webgpu: enabled<br>webnn: disabled_off|
|Load (avg)|undefined|
|Memory (System)|15.63GB (4.18GB free)|
|Process Argv|. --crash-reporter-id d0917b66-e997-4ad6-ab52-7668bb346569|
|Screen Reader|no|
|VM|0%|
</details><details><summary>Extensions (130)</summary>

Extension|Author (truncated)|Version
---|---|---
vscode-openapi|42C|4.32.2
zotenote|A-W|1.0.1
android-dev-ext|ade|1.4.0
Bookmarks|ale|13.5.0
openscad|Ant|1.3.1
spellright|ban|3.0.142
mermaid-markdown-syntax-highlighting|bpr|1.6.0
external-pdf|cha|1.2.0
doxdocgen|csc|1.4.0
vscode-markdownlint|Dav|0.53.0
vscode-eslint|dba|3.0.10
vscode-quick-select|dba|0.2.9
vscode-deno|den|3.43.6
gitlens|eam|14.6.1
EditorConfig|Edi|0.17.2
prettier-vscode|esb|11.0.0
figma-vscode-extension|fig|0.4.0
vscode-firefox-debug|fir|2.15.0
shell-format|fox|7.2.5
vscode-google-translate|fun|1.4.13
codespaces|Git|1.17.3
copilot|Git|1.294.1475
copilot-chat|Git|0.26.2025033102
copilot-workspace|git|0.5.4
remotehub|Git|0.64.0
vscode-github-actions|git|0.26.2
vscode-pull-request-github|Git|0.107.2025032704
geminicodeassist|goo|2.30.0
cloudcode|goo|2.28.0
overleaf-workshop|iam|0.15.0
cslpreview|igo|0.2.2
path-autocomplete|ion|1.25.0
latex-workshop|Jam|10.9.0
lilypond-syntax|jea|0.1.1
scheme|jea|0.2.0
better-cpp-syntax|jef|1.17.2
commitlint|jos|2.6.2
language-julia|jul|1.138.1
google-search|kam|0.0.1
vscode-lua-format|Koi|1.3.8
openscad-language-support|Lea|1.2.5
lilypond-formatter|lhl|0.2.3
lilypond-pdf-preview|lhl|0.2.8
lilypond-snippets|lhl|0.1.1
vslilypond|lhl|1.7.3
language-matlab|Mat|1.3.2
git-graph|mhu|1.30.0
mongodb-vscode|mon|1.12.1
azure-dev|ms-|0.8.4
vscode-azure-github-copilot|ms-|0.3.193
vscode-azureappservice|ms-|0.25.5
vscode-azurecontainerapps|ms-|0.6.1
vscode-azurefunctions|ms-|1.16.3
vscode-azureresourcegroups|ms-|0.8.3
vscode-azurestaticwebapps|ms-|0.12.2
vscode-azurestorage|ms-|0.16.3
vscode-azurevirtualmachines|ms-|0.6.7
vscode-cosmosdb|ms-|0.25.1
vscode-docker|ms-|1.29.4
vscode-dotnet-runtime|ms-|2.3.0
vscode-edge-devtools|ms-|2.1.8
black-formatter|ms-|2025.2.0
debugpy|ms-|2025.5.2025031101
flake8|ms-|2025.2.0
isort|ms-|2023.13.13171013
python|ms-|2025.3.2025032601
vscode-pylance|ms-|2025.3.104
jupyter|ms-|2025.3.2025032101
jupyter-keymap|ms-|1.1.2
jupyter-renderers|ms-|1.1.0
vscode-jupyter-cell-tags|ms-|0.1.8
vscode-jupyter-slideshow|ms-|0.1.5
remote-containers|ms-|0.408.0
remote-ssh|ms-|0.119.2025032715
remote-ssh-edit|ms-|0.87.0
remote-wsl|ms-|0.88.5
vscode-remote-extensionpack|ms-|0.25.0
azure-repos|ms-|0.40.0
cmake-tools|ms-|1.20.53
copilot-mermaid-diagram|ms-|0.0.2025032701
cpptools|ms-|1.24.4
cpptools-extension-pack|ms-|1.3.1
js-debug-nightly|ms-|2025.3.2117
live-server|ms-|0.5.2024091601
powershell|ms-|2025.3.0
remote-explorer|ms-|0.5.2024011009
remote-repositories|ms-|0.42.0
remote-server|ms-|1.6.2024011109
vscode-commander|ms-|0.2.0
vscode-copilot-data-analysis|ms-|0.2.2
vscode-github-issue-notebooks|ms-|0.0.130
vscode-node-azure-pack|ms-|1.2.0
vscode-serial-monitor|ms-|0.13.1
vscode-speech|ms-|0.12.1
vscode-speech-language-pack-en-ca|ms-|0.5.0
vscode-speech-language-pack-en-gb|ms-|0.5.0
vscode-speech-language-pack-ko-kr|ms-|0.5.0
vsliveshare|ms-|1.0.5948
windows-ai-studio|ms-|0.10.8
autodocstring|njp|0.6.1
pandocciter|not|0.10.4
typst-lsp|nva|0.13.0
publisher|pos|1.10.0
shiny|Pos|1.1.0
shinyuieditor|pos|0.5.0
quarto|qua|1.119.0
r-debugger|RDe|0.5.5
java|red|1.41.0
vscode-xml|red|0.28.0
vscode-yaml|red|1.14.0
r|REd|2.8.4
multi-command|ryu|1.6.0
vscode-deepl|soe|1.1.2
abc-music|sof|0.4.0
lua|sum|3.13.9
latex-utilities|tec|0.4.14
cmake|twx|0.0.17
vscode-terminal-here|Tyr|0.2.4
windows-terminal|Tyr|0.7.0
errorlens|use|3.16.0
intellicode-api-usage-examples|Vis|0.2.9
vscodeintellicode|Vis|1.2.30
vscode-gradle|vsc|3.16.4
vscode-java-debug|vsc|0.58.1
vscode-java-dependency|vsc|0.24.1
vscode-java-pack|vsc|0.29.0
vscode-java-test|vsc|0.43.0
vscode-maven|vsc|0.44.0
zotero-cite|XIN|0.8.9
markdown-all-in-one|yzh|3.6.3

(1 theme extensions excluded)

</details><details>
<summary>A/B Experiments</summary>

```
vsliv368cf:30146710
vspor879:30202332
vspor708:30202333
vspor363:30204092
pythonvspyt551cf:31249598
vscod805:30301674
py29gd2263:31024238
vscaac:30438845
c4g48928:30535728
962ge761:30841072
dsvsc014:30777825
dsvsc015:30821418
h48ei257:31000450
pythontbext0:30879054
cppperfnew:30980852
dwnewjupytercf:31046870
nativerepl1:31134653
pythonrstrctxt:31093868
nativeloc1:31118317
e80f6927:31120813
iacca1:31150324
dwcopilot:31158714
c3hdf307:31184662
6074i472:31201624
dwoutputs:31242946
customenabled:31248079
hdaa2157:31222309
copilot_t_ci:31222730
f27dg485:31264363
g012b348:31231168
jda6j935:31233686
copilothoveron:31262361
pythoneinst12:31251391
notreesitterini:31263227
bgtreat:31252123
fh1c7952:31247580
3ad0i337:31273073
31787653:31262186
3e8i5726:31271747
pythonpulldiag:31258010
6gi0g917:31259950
ceih0514:31264451
996jf627:31264550
pythonrdcb7:31268811
gh9da704:31272286

```

</details>

**Comment by meganrogge** [comment]
@jooyoungseo what you describe works for me. Which mode are you in when you encounter this issue?

**Comment by jooyoungseo** [comment]
@meganrogge Have you tested on Windows? It doesn't work in Chat panel, Edits, and any views.

**Comment by meganrogge** [comment]
I don't have access to windows at the moment, I have only tried Mac.

**Comment by meganrogge** [comment]
@jooyoungseo I just tested on windows and this is working for me. let's debug together on Friday.

**Comment by jooyoungseo** [comment]
@meganrogge  Yes, it's working again now.
